en/docs/monitoring-and-insights/alerts-overview.md

```diff
@@ -0,0 +1,145 @@
+
+Configurable Parameters:
+
+- Threshold: Requests per minute (e.g., 200).
+- Period: Monitoring window (e.g., 5 minutes).
+
+### Resources alerts
+
+Resource alerts notify you when your component’s CPU or memory usage exceeds the defined thresholds. This ensures you can fix the resources allocations early to avoid performance issues or downtimes.
+
+Configurable Parameters:
+
+- Metric: CPU or Memory.
+- Threshold: **mCPU** for CPU and **MiB** for Memory(e.g., 1000).
+- Period: Duration the threshold must be exceeded (e.g., 5 minutes).
+
+    !!! Tip
+        - **CPU**: mCPU (milliCPU) measures CPU usage in fractions of a core, where 1000m = 1 full core.
+        - **Memory**: MiB (Mebibyte) measures memory in binary units, where 1 MiB = 2^20^ bytes.
+
+### Logs alerts
+
+Logs alerts trigger notifications when a specific phrase appears frequently in your component logs. This helps to identify recurring issues or critical errors quickly, enabling faster troubleshooting.
+
+Configurable Parameters:
+
+- Search Phrase: Keyword or phrase (e.g., failed).
+- Count: Minimum occurrences to trigger the alert (e.g., 10).
+- Interval: Time window for counting occurrences (e.g., 5 minutes).
+
+### Build Failure alerts
+
+Build Failure alerts inform you if a build failure occurs for your component. This is essential for maintaining smooth development workflows, as it allows quick action to reduce downtime.
+
+### Status Code alerts
+
+Status Code alert triggers when your component returns specific HTTP error(s) (e.g., **403** Forbidden, **500** Internal Error). These alerts help to detect issues affecting your component’s availability or speed.
+
+Configurable Parameters:
+
+- Status Code: Error code or series (e.g., 400:Bad Request).
+- Count: Minimum occurrences (e.g., 5).
+- Interval: Time window (e.g., 5 minutes).
+
+!!! note
+    Currently Status Code alerts only supports api proxy component types.
+
+## Configure Alert
+
+If you are going to create an alert for the first time, follow the steps given below.
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/).
+2. Ensure you are in the correct organization where you have a project with the component to configure an alert.
+3. Navigate to the component by clicking on the project with the component to configure an alert.
+4. Click the component.
+
+    !!! info
+        make sure your component is deployed in order to make the alerts work.  
+
+5. In the Choreo Console left navigation menu, click **Observability**.
+6. In the left navigation menu on the **Observability** page, click **Alerts**. This opens the **Configure Alerts** pane by default.
+7. Click **Create Alert Rule** to create a new alert rule.
+
+    ![Create Alert Rule](../assets/img/monitoring-and-insights/alerts/alert-creation.png){.cInlineImage-full}
+
+    ![Create Alert Rule](../assets/img/monitoring-and-insights/alerts/alert-configuration.png){.cInlineImage-full}
+
```

Comment:
Let's add a note somewhere around this section to say that you can  configure a maximum of 10 alerts per component